scalability_and_performance/cnf-numa-aware-scheduling.adoc

```diff
@@ -37,6 +37,8 @@ include::modules/cnf-configuring-single-numa-policy.adoc[leveloffset=+2]
 
 .Additional resources
 
+* xref:../updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update.adoc#images-configuration-registry-mirror-configuring_updating-restricted-network-cluster[Configuring image registry repository mirroring]
```

Comment:
I think this link would work better immediately after the `[cnf-deploying-the-numa-aware-scheduler.adoc](https://github.com/openshift/openshift-docs/pull/84817/files#diff-696ffedffaa6c7a7e84d62ef21b03ff6d7e7bb1f08d18e11c545db8d79b6a7a3R34)` module.